tinyagentos/delegation_handler.py

```
@@ -0,0 +1,581 @@
+    # the agent's canonical_id and reset them.
+    try:
+        tasks = await task_store.list_for_assignee(canonical_id)
+    except AttributeError:
+        # Fall back: try list_tasks with filter.
+        try:
+            all_tasks = await task_store.list_tasks(
+                project_id=project_id,
+                status="in_progress",
+            )
+            tasks = [t for t in all_tasks if t.get("assignee_id") == canonical_id]
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)